### PR TITLE
test: add node value position lookup tests

### PR DESCRIPTION
**Pull request by byCedric**
This might help if snapshots are failing on node positions. It checks if the `node.value` matches the value from `line:column` and `offset` lookups.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 const { describe, beforeEach, it } = require('mocha')
-const { expect } = require('chai')
+const { expect, assert } = require('chai')
```
Comment (byCedric) [inline test.js:2]: Couldn't get `expect().to.be` working, kept failing `.to` or `.be` is not a function.

Comment (bcoe) [reply test.js:2]: That's odd, but I tend to prefer `assert` any ways, mainly pulled in `expect` for its `throw` syntax ...

Perhaps we could switch to [Node's built in throw helper](https://nodejs.org/api/assert.html#assert_assert_throws_fn_error_message) and drop our use of `expect` entirely, using `chai` only for the snapshots?

File: test.js [hunk 2]
```diff
@@ -84,5 +95,18 @@ describe('<message>', () => {
...
+    it('contains valid positions for footer after multi-line body', () => {
+      assertNodePositions('fix: address major bug\n\nthis is the first line of the body\n\nthis is the second line of body\n\nAuthor: @bcoe\nRefs #392')
+    })
   })
 })
+
+function assertNodePositions (text) {
+  visit(parser(text), node => {
+    if (node.children) return
+    const valueByPoint = source(node, text)
```
Comment (bcoe) [inline test.js:107]: For nodes where we've dropped whitespace information, will `source` fail today? This seems like a much better approach for getting the original header than I'd used in #19, perhaps I can move to this approach after we land this?

**Review by bcoe — APPROVED** [review]
LGTM, with the nit that perhaps we could drop use of `expect` entirely.

File: test.js (hunk 1 above)
Comment (byCedric) [reply test.js:2]: TIL node has its own assert 😄

File: test.js (hunk 2 above)
Comment (byCedric) [reply test.js:107]: As long as we don't include that in the positional information, should be fine. I'm not sure if that's the correct approach, we still remove data from the original content. But, that piece of info is not part of that node.

**Pull request merged**